### PR TITLE
#4282 fix: duplicated contact search result

### LINKED ISSUE #4282: Duplicate contact search result
The test `compose - can load contact based on name` passes, but `human@flowcrypt.com` is displayed twice by the end of the test
image
I would expect it to appear only once

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR fixed duplicated contact search result

close #4282 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Test updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Review by sosnovsky — COMMENTED** [review]
Looks good, but let's also update ui test `compose - can load contact based on name` to ensure that search results list contains only 1 contact

File: test/source/browser/controllable.ts [hunk 1]
```diff
@@ -523,6 +523,11 @@ abstract class ControllableBase {
     return files;
   };
 
+  public checkElementCount = async (selector: string, expectedCount: number) => {
```
Comment (sosnovsky) [inline test/source/browser/controllable.ts:526]: There is already [ensureElementCount](https://github.com/FlowCrypt/flowcrypt-browser/blob/feature/issue-4282/test/source/browser/controllable.ts#L464) method which performs the same check. Let's merge them into one to remove duplication

Comment (ioanmo226) [reply test/source/browser/controllable.ts]: You are right 👍 
Just merged it

**Review by sosnovsky — APPROVED** [review]
👍

**Pull request merged**